sway/input/seat.c

```diff
@@ -402,6 +402,31 @@ static void drag_icon_handle_destroy(struct wl_listener *listener, void *data) {
 	free(icon);
 }
 
+static void drag_handle_destroy(struct wl_listener *listener, void *data) {
+	struct sway_drag *drag = wl_container_of(listener, drag, destroy);
+
+	// Focus enter isn't sent during drag, so refocus the focused node, layer
+	// surface or unmanaged surface.
```

Comment:
Would it be more appropriate to call `cursor_rebase` here?

I guess maybe that'd only work if focus-follows-mouse is enabled?

Reply:
Won't that only affect the cursor? The cursor *is* affected by this issue, but the biggest problem is keyboard focus, which requires a full leave and re-enter to fix.

Reply:
Right, I see, this makes sense.